seed/views/columns.py

```diff
@@ -149,6 +149,20 @@ def retrieve(self, request, pk=None):
             'column': ColumnSerializer(c).data
         })
 
+    @ajax_request_class
+    @has_perm_class('can_modify_data')
+    def update(self, request, pk=None):
+        organization_id = request.query_params.get('organization_id', None)
+
+        request.data['shared_field_type'] = request.data['sharedFieldType']
+        del request.data['sharedFieldType']
```

Comment:
@nllong Why is it necessary to do this?  Seems like it comes from here:
https://github.com/SEED-platform/seed/blob/7c539d722609b296a676953f48a9f96633a936cd/seed/models/columns.py#L1247-L1248